.circleci/config.yml

```diff
@@ -19,9 +19,16 @@ jobs:
       - checkout
       - run: make unit-test
       - run: bash <(curl -s https://codecov.io/bash)
+  bddtests:
+    docker:
+      - image: circleci/golang:1.12
+    steps:
+      - checkout
+      - run: make bddtests
 workflows:
   version: 2
   checks-test:
     jobs:
       - checks
       - unit-test
+      - bddtests
```

Comment:
prefer we are consistent in naming. the job above is "unit-test" so I would assume this would be "bdd-test".

Reply:
done